files/jcl/ZWESECUR.jcl

```diff
@@ -1,488 +1,442 @@
-
-/* permit Zowe main server to use ZSS, cross memory server           */
-  RLIST   FACILITY ZWEX.IS ALL
-  RDEFINE FACILITY ZWEX.IS UACC(NONE)
-  PERMIT ZWEX.IS CLASS(FACILITY) ACCESS(READ) ID(&ZOWEUSER.)
-
-  SETROPTS RACLIST(FACILITY) REFRESH
-
-/** uncomment to use SUPERUSER.FILESYS, see JCL comments             */
-/** permit Zowe main server to write persistent data                 */
-/* RLIST   UNIXPRIV SUPERUSER.FILESYS ALL                            */
-/* RDEFINE UNIXPRIV SUPERUSER.FILESYS UACC(NONE)                     */
-/* PERMIT SUPERUSER.FILESYS CLASS(UNIXPRIV) ACCESS(CONTROL) -        */
-/*  ID(&ZOWEUSER.)                                                   */
-
-/* SETROPTS RACLIST(UNIXPRIV) REFRESH                                */
-
-/* show results .................................................... */
-  RLIST   FACILITY ZWEX.IS           ALL
-  RLIST   UNIXPRIV SUPERUSER.FILESYS ALL
-
-/* DEFINE ZSS SERVER PERMISIONS .................................... */
-
-/* permit ZSS to create a user's security environment                */
-/* ATTENTION: Defining the BPX.DAEMON or BPX.SERVER profile makes    */
-/*            z/OS UNIX switch to z/OS UNIX level security, which is */
-/*            more secure, but it can impact operation of existing   */
-/*            applications. Test this thoroughly before activating   */
-/*            it on a production system.                             */
-  RLIST   FACILITY BPX.DAEMON ALL
-  RDEFINE FACILITY BPX.DAEMON UACC(NONE)
-  PERMIT BPX.DAEMON CLASS(FACILITY) ACCESS(UPDATE) ID(&ZSSUSER.)
-
-  RLIST   FACILITY BPX.SERVER ALL
-  RDEFINE FACILITY BPX.SERVER UACC(NONE)
-  PERMIT BPX.SERVER CLASS(FACILITY) ACCESS(UPDATE) ID(&ZSSUSER.)
-
-  SETROPTS RACLIST(FACILITY) REFRESH
-
-/* show results .................................................... */
-  RLIST   FACILITY BPX.DAEMON ALL
-  RLIST   FACILITY BPX.SERVER ALL
-
-/* DEFINE ZOWE DATA SET PROTECTION ................................. */
-
-/* - &HLQ..SZWEAUTH is an APF authorized data set. It is strongly    */
-/*   advised to protect it against updates.                          */
-/* - The sample commands assume that EGN (Enhanced Generic Naming)   */
-/*   is active, which allows the usage of ** to represent any number */
-/*   of qualifiers in the DATASET class. Substitute ** with * if EGN */
-/*   is not active on your system.                                   */
-
-/* HLQ stub                                                          */
-  LISTGRP  &HLQ. ALL
-  ADDGROUP &HLQ. DATA('Zowe - HLQ STUB')
-
-/* general data set protection                                       */
-  LISTDSD PREFIX(&HLQ.) ALL
-  ADDSD  '&HLQ..*.**' UACC(READ) DATA('Zowe')
-  PERMIT '&HLQ..*.**' CLASS(DATASET) ACCESS(ALTER) ID(&SYSPROG.)
-
-  SETROPTS GENERIC(DATASET) REFRESH
-
-/* show results .................................................... */
-  LISTGRP &HLQ.         ALL
-  LISTDSD PREFIX(&HLQ.) ALL
-
```

Comment:
Why have you deleted Onno's new changes!?

Reply:
I've re-instated Onno's new changes.